docs/dev/bilrost-migration-guidelines.md

```diff
@@ -0,0 +1,244 @@
+# Bilrost best practice
+
+So I have been working with bilrost the last couple of days mainly on network serialization. I also did some work on the storage encoding specially the `Envelope` structure.
```

Comment:
That won't stand well against the test of time. Perhaps frame the document as a functional guide on our recommendations and less on your personal experience.